CMakeLists.txt

```diff
@@ -204,6 +204,11 @@ google_binary(cartographer_pbstream
   cartographer/io/pbstream_main.cc
 )
 
+google_binary(cartographer_print_configuration
+  SRCS
+  cartographer/common/configuration_printer_main.cc
```

Comment:
move and rename to common/internal/print_configuration_main.cc
(if we had a dev/ folder like in cartographer_ros I'd put it there but also I wouldn't create one for a single tool.)